benchmarks/src/garage_benchmarks/helper.py

```diff
@@ -72,8 +72,12 @@ def wrapper_func():
         plt.close('all')
 
         _log_dir = _get_log_dir(exec_func.__name__)
+
         if os.path.exists(_log_dir):
-            shutil.rmtree(_log_dir)
+            count = 1
+            while os.path.exists(_log_dir + '(' + str(count) + ')'):
```

Comment:
maybe make this path a little easier to type on the terminal (i.e. no (), which have to be escaped)